graph-commons/src/test/scala/io/renku/http/client/GitLabClientSpec.scala

```diff
@@ -51,99 +50,131 @@ class GitLabClientSpec
     with should.Matchers
     with MockFactory {
 
-  "send" should {
+  "get" should {
 
     "fetch json from a given page" in new TestCase {
 
       stubFor {
-        callMethod(method, s"/api/v4/$path")
+        callMethod(GET, s"/api/v4/$path")
           .willReturn(
             okJson(result)
               .withHeader("X-Next-Page", maybeNextPage.map(_.show).getOrElse(""))
           )
       }
 
-      client.send(method, path, endpointName)(mapResponse).unsafeRunSync().toString() shouldBe result
+      client.get(path, endpointName)(mapGetResponse).unsafeRunSync().toString() shouldBe result
     }
 
     "fetch commits using the given access token" in new TestCase {
       implicit val accessToken = accessTokens.generateOne.some
       stubFor {
-        callMethod(method, s"/api/v4/$path")
+        callMethod(GET, s"/api/v4/$path")
           .withAccessToken(accessToken)
           .willReturn(
             okJson(result)
               .withHeader("X-Next-Page", maybeNextPage.map(_.show).getOrElse(""))
           )
       }
 
-      client.send(method, path, endpointName)(mapResponse).unsafeRunSync().toString() shouldBe result
+      client.get(path, endpointName)(mapGetResponse).unsafeRunSync().toString() shouldBe result
     }
 
     "Handle empty JSON" in new TestCase {
```

Comment:
For this test and the next 404 one shouldn't we maybe say smth the outcome of the `get` method depends on the given mapping? I mean, in the general case of the GitLabClient, it's not entirely true if the client will always handle an empty JSON or 404. The client will do what the mapping you pass to it does. Maybe we can even collapse these two tests into one more abstract.